dali/operators/sequence/optical_flow/optical_flow.cc

```diff
@@ -109,34 +126,33 @@ void OpticalFlow<GPUBackend>::RunImpl(Workspace<GPUBackend> &ws) {
     CUDA_CALL(cudaEventRecord(sync_, ws.stream()));
     CUDA_CALL(cudaStreamWaitEvent(of_stream, sync_, 0));
   }
+  const auto &input = ws.Input<GPUBackend>(0);
+  auto &output = ws.Output<GPUBackend>(0);
+  output.SetLayout("HWC");  // Channels represent the two flow vector components (x and y)
+
+  auto input_sh = input.shape();
+
+  // sort all sequences by size, to make sure that samples are processed grouped by shape to avoid
+  // NV OF reconfiguration as  much as possible
+  std::vector<DimsOrder> processing_order;
```

Comment:
Okay, I got the answer below.:)

Reply:
It was to avoid frequent configurations. I haven't compared perf compared to the order. I would expect rather negligible perf difference, and this code it to avoid expensive OF reconfiguration (deallocation of the buffers, destroying driver structures and recreation of it).